src/policy/sft_map.rs

```diff
@@ -150,36 +154,37 @@ mod space_map {
         /// Currently our spaces are using address range 0x0000_0200_0000_0000 to 0x0000_2200_0000_0000 (with a maximum of 16 spaces).
         /// When masked with this constant, the index is 1 to 16. If we mask any arbitrary address with this mask, we will get 0 to 31 (32 entries).
         pub const ADDRESS_MASK: usize = 0x0000_3f00_0000_0000usize;
```

Comment:
We need to fix this as well right? An example is for sv39, we might still have the same number of spaces, but each space is smaller.

Reply:
Yes. This should be calculated from heap start and space extent.

Reply:
Done